sensors/test_deps.go

```diff
@@ -119,10 +117,30 @@ func getInvalidSensor() *inject.Camera {
 	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
+	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
+		return camera.Properties{SupportsPCD: true}, nil
+	}
+	return cam
+}
+
+func getNoPCDCamera() *inject.Camera {
```

Comment:
this is unused right?

Reply:
please remove code changes not needed to implement the acceptance criteria of the ticket, or comment them out with a link to the ticket they will be for